lib/sycamore/README.md

```diff
@@ -0,0 +1,63 @@
+![SycamoreLogoFinal.svg](https://raw.githubusercontent.com/aryn-ai/sycamore/main/docs/source/images/sycamore_logo.svg)
+
+[![PyPI](https://img.shields.io/pypi/v/sycamore-ai)](https://pypi.org/project/sycamore-ai/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/sycamore-ai)](https://pypi.org/project/sycamore-ai/)
+[![Slack](https://img.shields.io/badge/slack-sycamore-brightgreen.svg?logo=slack)](https://join.slack.com/t/sycamore-ulj8912/shared_invite/zt-23sv0yhgy-MywV5dkVQ~F98Aoejo48Jg)
+[![Docs](https://readthedocs.org/projects/sycamore/badge/?version=stable)](https://sycamore.readthedocs.io/en/stable/?badge=stable)
```

Comment:
Do the docs still work? I'm wondering if the autodoc stuff that creates API docs from docstrings will still pick up the library. At the very least, I suspect the paths need to be updated in this gen script: https://github.com/aryn-ai/sycamore/blob/main/docs/source/APIs/gen

Reply:
That's a very good question.

Reply:
surprisingly it seems like running a `make html` correctly generated all the docs... updating paths to sycamore lib anyway